src/rpc/blockchain.cpp

```diff
@@ -593,12 +593,16 @@ static CBlockUndo GetUndoChecked(BlockManager& blockman, const CBlockIndex* pblo
 {
     AssertLockHeld(::cs_main);
     CBlockUndo blockUndo;
-    if (blockman.IsBlockPruned(pblockindex)) {
-        throw JSONRPCError(RPC_MISC_ERROR, "Undo data not available (pruned data)");
-    }
 
-    if (!UndoReadFromDisk(blockUndo, pblockindex)) {
-        throw JSONRPCError(RPC_MISC_ERROR, "Can't read undo data from disk");
+    // Ignore genesis block
+    if (pblockindex->nHeight > 0) {
```

Comment:
to minimize the changes to a single line, could just return early if height is 0.
e.g.
```c++
// Ignore genesis block
if (pblockindex->nHeight == 0) return {};

CBlockUndo blockUndo;
// etc..
```